src/TextField/TextField.js

```diff
@@ -2,9 +2,19 @@
 
 import React, { Component, PropTypes } from 'react';
 import shallowEqual from 'recompose/shallowEqual';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
 import { Input, InputLabel } from '../Input';
 import FormControl from '../Form/FormControl';
 
+export const styleSheet = createStyleSheet('TextField', () => {
+  return {
+    root: {
+      marginTop: 16,
```

Comment:
> TextField is higher order than the simple inputs. 

I believe that the form layout should be handled by an even higher-level component.
Not that one. At the office, we use the `<Layout />` for our forms
I'm pretty convinced that we should remove it. Or at least having an option opt-out.